making-programming-videos/README.md

```diff
@@ -0,0 +1,32 @@
+# How to Make Engaging Programming Videos
+
+This repository provides the VSCode settings file that is mentioned in the Real Python [How to Make Engaging Programming Videos](https://realpython.com/making-programming-videos/) tutorial.
+
+## Using the Settings
+
+The provided settings file helps you to have a clutter-free and well-readable VSCode editor.
+You can either replace or enhance your user settings file with the contents of [settings.json](settings.json).
+
+Depending on your platform, you can find your user settings file here:
```

Comment:
Hm, I'm not sure it makes sense to suggest them to use this file as their overall user settings 🤔 - They'll probably want more screen space when coding on a project themselves.

I usually pop the file into the project I'm working on, in the `.vscode` folder:

```
project/
│
├── .vscode/
│   └── settings.json
│
└── code.py
```

That applies it just for the project you're working on.

Unless you're thinking that they're using a separate account for recording? IDK, maybe give some more context, or otherwise less information and just provide two links on how they can set it either on a user-level or a project-level.